avionics/common/include/radio.h

```diff
@@ -189,7 +192,14 @@ class Radio {
 
     static void sendConfig(const uint32_t time);
 
-  private:
```

Comment:
wait was the `private:` specifier not put back?

Reply:
Sorry for the late reply but yes, the private specifier is still there.